### PR TITLE
K8

### PR DESCRIPTION
**Pull request by kinuesantos**

### CODE REVIEW

File: lesson4/practice/questions.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Create a function:
```
Comment (seemcat) [inline lesson4/practice/questions.txt:1]: You don't have to include this file in your Pull Request. Delete.

File: lesson4/public/index28.html [hunk 2]
```diff
@@ -34,6 +58,7 @@
   xhttp.open('GET', `/storing.txt`);
   xhttp.onreadystatechange = ()=>{
     if(xhttp.responseText){
+      console.log('xhttp.responseText',xhttp.responseText);
```
Comment (seemcat) [inline lesson4/public/index28.html:61]: Delete your console.log

Comment (kinuesantos) [reply lesson4/public/index28.html:61]: done

Comment (kinuesantos) [reply lesson4/public/index28.html:61]: thanks!

**Review by chocobrownie — APPROVED** [review]

**Review by MadhuBeesetty — APPROVED** [review]
looks good...

**Pull request merged**